eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Enact.hs

```diff
@@ -82,8 +81,8 @@ enactmentTransition = do
           wdrlsNoNetworkId = Map.mapKeys getRwdCred wdrls
       pure
         st
-          { ensWithdrawals = Map.unionWith (<>) wdrlsNoNetworkId $ ensWithdrawals st
-          , ensTreasury = ensTreasury st <-> wdrlsAmount
+          { ensWithdrawals = Map.unionWith (+) wdrlsNoNetworkId $ ensWithdrawals st
+          , ensTreasury = ensTreasury st - wdrlsAmount -- TODO: This can be removed as it is an unused field
```

Comment:
Let me know whether you think this is correct, so I will go ahead and remove this field.

Reply:
This is wrong indeed.